packages/lesswrong/components/forumEvents/givingSeason/GivingSeason2025Banner.tsx

```diff
@@ -0,0 +1,462 @@
+                  sortKeyType="Date"
+                  renderers={{
+                    newPost: {
+                      fragmentName: "PostsList",
+                      render: (post: PostsList) => (
+                        <GivingSeasonFeedItem
+                          href={postGetPageUrl(post)}
+                          iconStyle="post"
+                          action="posted"
+                          user={post.user}
+                          post={post}
+                          date={post.postedAt}
+                          preview={post.contents?.plaintextDescription ?? ""}
+                        />
+                      ),
+                    },
+                    newComment: {
+                      fragmentName: "CommentsListWithParentMetadata",
+                      render: (comment: CommentsListWithParentMetadata) => (
+                        <GivingSeasonFeedItem
+                          href={commentGetPageUrl(comment)}
+                          iconStyle="comment"
+                          action="on"
+                          user={comment.user}
+                          post={comment.post}
+                          date={comment.postedAt}
+                          preview={comment.contents?.plaintextMainText ?? ""}
+                        />
+                      ),
+                    },
+                  }}
+                />
+            )}
+            {selectedEvent?.tag &&
+                selectedEvent.end < now &&
+                selectedEvent !== currentEvent && (
+              <GivingSeasonTopPosts
+                tagId={selectedEvent.tag._id}
+                tagSlug={selectedEvent.tag.slug}
+              />
+            )}
+          </div>
+        </div>
+        <div className={classes.election}>
+          <div className={classes.electionStatus}>
+            <div className={classes.amountRaised}>
+              ${formatStat(amountRaised)} raised{" "}
+              <span>to the Donation Election Fund</span>
+            </div>
+            <div className={classes.progress} aria-hidden>
+              <div className={classes.progressBackground} />
+              <div
+                style={{width: `${amountRaised / amountTarget * 100}%`}}
+                className={classes.progressBar}
+              />
+            </div>
+            <div className={classes.match}>
+              We&apos;re matching the first $5000
+            </div>
+          </div>
+          <div className={classes.electionButtons}>
+            <button
+              onClick={onButtonClick.bind(null, "learnMore", ELECTION_INFO_HREF)}
+              className={classes.buttonOutlined}
+            >
+              Learn more
+            </button>
+            <button
+              onClick={onButtonClick.bind(null, "donate", ELECTION_DONATE_HREF)}
+              className={classes.buttonFilled}
+            >
+              Donate
+            </button>
```

Comment:
Could these be `<a>`s rather than buttons so that you can right click to copy the link? Also I would personally prefer these to open in a new tab as I feel like donating at least is a fairly big action that you may want to start and then come back to later (cc @agnesstenlund )